src/Schema.js

```diff
@@ -433,6 +432,39 @@ class Schema {
 		});
 	}
 
+	collectSingleInterfaceDependencies(schema, interfaceName) {
+		let interfaces = this.schemas[this.defaultSchemaName].interfaces;
+		interfaces = interfaces.map(i => i.name);
```

Comment:
Wouldn't `interfaces` be an object type?

Reply:
Oh.. :P,
Fixing it